apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/bigDecimalAssertions.kt

```diff
@@ -0,0 +1,153 @@
+@file:Suppress("JAVA_MODULE_DOES_NOT_READ_UNNAMED_MODULE" /* TODO remove once https://youtrack.jetbrains.com/issue/KT-35343 is fixed */)
+
+package ch.tutteli.atrium.api.infix.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.domain.builders.bigDecimal
+import ch.tutteli.atrium.domain.builders.creating.PleaseUseReplacementException
+import java.math.BigDecimal
+
+/**
+ * Deprecated as it would compare the subject against [expected] including scale
+ * -- many developers are not aware of that.
+ *
+ * Use [isNumericallyEqualTo] if you expect that the following assertion holds:
+ * ```
+ * expect(BigDecimal("10").toBe(BigDecimal("10.0"))
+ * ```
+ * However, if you expect it to be wrong (because `BigDecimal.scale` differ), then use [isEqualIncludingScale].
+ */
+@Deprecated(
+    "Use `isNumericallyEqualTo` if you expect that the following assertion holds:\n" +
+        "`expect(BigDecimal(\"10\")).toBe(BigDecimal(\"10.0\"))`\n" +
+        "However, if you expect it to be wrong (because `BigDecimal.scale` differ), then use `isEqualIncludingScale`.",
+    ReplaceWith("isNumericallyEqualTo(expected) or isEqualIncludingScale(expected)")
+)
+@Suppress("UNUSED_PARAMETER", "unused")
+infix fun <T : BigDecimal> Expect<T>.toBe(expected: T): Nothing = throw PleaseUseReplacementException(
+    "BigDecimal.equals() compares also BigDecimal.scale, which you might not be aware of.\n" +
+        "If you know it and want that `scale` is included in the comparison, then use `isEqualIncludingScale`."
+)
+
+@Suppress("UNUSED_PARAMETER", "unused")
+@JvmName("toBeNullable")
+@Deprecated(
+    "Use `isNumericallyEqualTo` if you expect that the following assertion holds:\n" +
+        "`expect(BigDecimal(\"10\")).toBe(BigDecimal(\"10.0\"))`\n" +
+        "However, if you expect it to be wrong (because `BigDecimal.scale` differ), then use `isEqualIncludingScale`.",
+    ReplaceWith("isNumericallyEqualTo(expected) or isEqualIncludingScale(expected)")
+)
+infix fun <T : BigDecimal?> Expect<T>.toBe(expected: T): Nothing = throw PleaseUseReplacementException(
+    "BigDecimal.equals() compares also BigDecimal.scale, which you might not be aware of.\n" +
+        "If you know it and want that `scale` is included in the comparison, then use `isEqualIncludingScale`."
+)
+
+/**
+ * Expects that the subject of the assertion (a [BigDecimal]) is `null`.
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.11.0
```

Comment:
please remove the ` @since` this function exists already some time, we started to add the `@since`  tag in v0.9.0